mobile/cmp/dialog/Dialog.scss

```diff
@@ -15,7 +19,7 @@
     .dialog-container {
       display: flex;
       flex-direction: column;
-      max-width: calc(100vw - var(--xh-pad-double-px));
+      max-width: calc(100% - var(--xh-pad-double-px));
```

Comment:
Please review the comments above re. "vh units are unreliable" - this is confusing how it's left - should have holistic approach to "here is how we manage issues around viewport and safe areas" for dialogs